### ISSUE TITLE
Sports dataset on the test set how to get the results in the paper

### ISSUE THREAD

**Issue by congyao123456** [MCG-NJU/MixSort#9]
Hello Dear Author:
       Thank you for your contribution to the MOT community, I have some questions about how to get the effect of mix ocsort in the paper in the sports test set, do I need to use the weights XX_sports_mix.pth.tar or XX_sports_train.pth.tar of which yolox and mixformer you have provided for training or XX_sports_train. pth.tar?, and some specific hyperparameters, looking forward to your reply!

**Comment by ret-1** [comment]
Hello, 

You need to use the provided corresponding weights to get the results. For hyperparameters, you can just use the default values in https://github.com/MCG-NJU/MixSort/blob/main/tools/track_mixsort_oc.py
